vulnfeeds/cmd/cperepos/Dockerfile

```diff
@@ -0,0 +1,41 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM ubuntu:22.04
```

Comment:
Could we instead use gcr.io/google.com/cloudsdktool/google-cloud-cli:alpine ? 

(from https://github.com/GoogleCloudPlatform/cloud-sdk-docker). 

This should be much smaller, and still include gsutil.

Reply:
Done in [0fd5da6](https://github.com/google/osv.dev/pull/1146/commits/0fd5da63efcf108cccf1714a78d75cc54769cdb8)